packages/flutter/lib/src/services/raw_keyboard_macos.dart

```diff
@@ -197,6 +202,23 @@ class RawKeyEventDataMacOs extends RawKeyEventData {
     return null;
   }
 
+  /// Returns true if the given label represents a function key.
+  ///
+  /// Examples of function characters are characters like "NSUpArrowFunctionKey = 0xF700"
+  /// or "NSHomeFunctionKey = 0xF729".
+  ///
+  /// See <https://developer.apple.com/documentation/appkit/1535851-function-key_unicodes?language=objc> for more
+  /// information.
+  ///
+  /// Used by [RawKeyEvent] subclasses to help construct IDs.
+  static bool isFunctionKey(String label) {
```

Comment:
`isFunctionKey` is misleading, since it includes things other than `F1`, `F2`, etc.  Maybe call it `isUnprintableKey` or some other name?

Reply:
Done. Also made it private.

Reply:
It doesn't seem to have changed...